sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Span, SpanOptions, SpanKind } from "@opentelemetry/api";
+import { getTracer } from "../src/tracerProxy";
+
+/**
+ * Tracing options to set on an operation.
+ * @hidden
+ */
+export interface OperationTracingOptionsLike {
+  /**
+   * OpenTelemetry SpanOptions used to create a span when tracing is enabled.
+   */
+  spanOptions?: SpanOptions;
+}
+
+/**
+ * Configuration for creating a new Tracing Span
+ * @hidden
+ */
+export interface SpanConfig {
+  /**
+   * Package name prefix
+   */
+  packagePrefix: string;
+  /**
+   * Service namespace
+   */
+  namespace: string;
+}
+
+/**
+ * Creates a function called createSpan to create spans using the global tracer.
+ * @hidden
```

Comment:
not really sure why this is hidden either

Reply:
This might not be necessary. @deyaaeldeen mentioned that one of the interesting bits with `hidden` is that you could use it to "hide" something that was actually exported from the index.ts (unlike an `internal` function, which is probably just exported so you can use it internally in the same package).

However, maybe we don't need to even bother with that since this function is public but part of our "preview" surface. Let me know either way.

Reply:
Oh that's a good idea actually. Let's keep it